adafruit_esp32spi/adafruit_esp32spi.py

```diff
@@ -563,17 +563,23 @@ def connect(self, secrets):
         that contains a 'ssid' and 'password' entry"""
         self.connect_AP(secrets["ssid"], secrets["password"])
 
-    def connect_AP(self, ssid, password, timeout_s=10):  # pylint: disable=invalid-name
+    def connect_AP(
+        self, ssid, password, timeout_s=10, show_password=False
```

Comment:
I'd rather not have the kwarg here. I think it should go next to the `debug` flag. It could be `debug_show_secrets` to cover other sorts of keys in addition to the password.

Reply:
Good thinking! To be clear, you're suggesting it should be an __init__ param?